src/Compiler/CompilerOptions.cpp

```diff
@@ -0,0 +1,257 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------------------------ CompilerOptions.cpp -------------------------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// Functions for adding options.
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/Support/Debug.h"
+
+#include "ExternalUtil.hpp"
+#include "onnx-mlir/Compiler/OMCompilerTypes.h"
+#include "src/Compiler/CompilerOptions.hpp"
+
+#define DEBUG_TYPE "compiler_options"
+
+using namespace std;
+using namespace onnx_mlir;
+
+llvm::cl::OptionCategory OnnxMlirOptions(
```

Comment:
Possible to sink this declaration inside the `onnx_mlir` namespace?

Reply:
Done. Now they are inside the `onnx_mlir` namespace.